web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1434,10 +1437,23 @@ public void run() {
 							        || wizardModel.databaseConnection.contains("127.0.0.1")) {
 								host = "'localhost'";
 							}
-							String sql = "drop user '?'@" + host;
+							
+							String sql = "";
+							if (wizardModel.databaseConnection.contains("mysql")) {
```

Comment:
Shouldn't we be using a constanct for mysql or posgresql?

Reply:
Or even better, wrap `wizardModel.databaseConnection.contains` in a method?

Reply:
I'll make use of constants for db's within this same class as private members and wrap wizardModel in a method like :
```
public boolean checkForDatabase( InitializationWizardModel wizardModel ){
            if( wizardModel.databaseConnection.contains(MYSQL) )
                   return true;
           else if( wizardModel.databaseConnection.contains(POSTGRESQL)
                     return true;
           else ...... for other db's

          return false;
}
```

Reply:
@dkayiwa , I have made some changes according to your suggestion.